### PR TITLE
feat: integrate better-n

### PR DESCRIPTION
**Pull request by bmrips**
I think that the [better-n plugin](https://github.com/jonatan-branting/nvim-better-n) aligns well with this plugin, hence I've added cross-integration. What do you think?

### CODE REVIEW

File: lua/impairative/operations.lua [hunk 1]
```diff
@@ -221,12 +248,7 @@ function ImpairativeOperations:command_pair(args)
             if 0 < vim.v.count then
                 cmd = vim.v.count .. cmd
             end
-            local ok, result = pcall(vim.cmd, cmd)
-            if not ok then
-                result = tostring(result)
-                result = result:match"E%d*:.+" or result
-                vim.api.nvim_err_writeln(result)
-            end
+            vim.cmd(cmd)
```
Comment (idanarye) [inline lua/impairative/operations.lua]: Why did you make this change? It seems unrelated to the better-n integration.

File: lua/impairative/operations.lua [hunk 2]
```diff
@@ -51,9 +51,26 @@ local function process_desc(desc, i)
     end
 end
 
+local function maybe_with_better_n(args, callbacks)
+    if not args.better_n then
+        return callbacks
+    else
+        print('better-n integration enabled for ' .. args.key)
```
Comment (idanarye) [inline lua/impairative/operations.lua]: Leftovers?

Comment (bmrips) [reply lua/impairative/operations.lua]: Yes! Fix incoming.

File: lua/impairative/operations.lua (hunk 1 above)
Comment (bmrips) [reply lua/impairative/operations.lua]: Leftovers from debugging

**Comment by bmrips** [comment]
Thank you! :heart:
